Squablr/View Controllers/EditProfileViewController.m

```diff
@@ -6,8 +6,10 @@
 //
 
 #import "EditProfileViewController.h"
+#import "Profile.h"
 
 @interface EditProfileViewController ()
+@property (nonatomic, strong) NSMutableArray *userProfileInfo;
```

Comment:
Why is userProfileInfo an array?

Reply:
I found an array suitable for storing the user object that way I can access all of their profile information but since it is only one user object I could make it a different data type 🤔